aldryn_reversion/test_helpers/test_app/models.py

```diff
@@ -44,7 +44,7 @@ class WithPlaceholder(models.Model):
     """
     Simple model with PlaceholderField.
     """
-    content = PlaceholderField('Helper model Content')
+    content = PlaceholderField('WithPlaceholder Content ph')
```

Comment:
its a class name, just to easy the debug process, though i'm not sure that ``cms.models.Placeholder`` ``slot`` field can contain spaces.